### PR TITLE
Bug1671123 5.7

### PR DESCRIPTION
**Pull request by laurynas-biveinis**
http://jenkins.percona.com/job/mysql-5.7-param/712/

### CODE REVIEW

File: mysql-test/suite/innodb/t/innodb-wl6445.test [hunk 1]
```diff
@@ -79,24 +63,8 @@ DROP DATABASE WL6445;
...
-
---echo # Restart server.
---exec echo "restart:" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
-
---enable_reconnect
---source include/wait_until_connected_again.inc
---disable_reconnect
-
---echo # Restarted
+let $restart_parameters = restart;
```
Comment (percona-ysorokin) [inline mysql-test/suite/innodb/t/innodb-wl6445.test:66]: use '--let $restart_parameters=' instead?

**Review by percona-ysorokin — APPROVED** [review]
LGTM
(minor comment can be ignored)

File: mysql-test/suite/innodb/t/innodb-wl6445.test (hunk 1 above)
Comment (laurynas-biveinis) [reply mysql-test/suite/innodb/t/innodb-wl6445.test:66]: Since this is a pristine 8.0 backport, I'd leave it untouched if possible

**Review by robgolebiowski — APPROVED** [review]
LGTM

**Pull request merged**